### ISSUE TITLE
PI - New features are navigable by keyboard.

### ISSUE THREAD

**Issue by kimberlythegeek** [mozilla-lockbox/lockbox-extension#538]
_< 3 hours_
Navigate the web content using only the keyboard
- Tab Order: `TAB`, as well as `SHIFT + TAB`, follows a logical and intuitive order.
- All controls, links, buttons, etc., get focused.
- Focus should be visibly apparent.
- When a modal or pop up window opens, focus shifts to the pop up.
  - Focus and tab order is constrained within the modal.
  - Modal can be exited via keyboard.
  - When a pop up window is closed, focus returns to a logical point.
- All content that is visually hidden is also hidden from the keyboard and/or a screen reader. _Except content specifically for screen reader users._
- Firefox toolbar icon can be accessed by keyboard and screen reader users.

**Comment by kimberlythegeek** [comment]
Manually tested that latest features are navigable by keyboard.
